.github/workflows/pathogen-repo-build.yaml

```diff
@@ -274,14 +275,14 @@ jobs:
           # shellcheck disable=SC2154
           set -x
 
-          eval "$NEXTSTRAIN_BUILD_COMMAND" |& tee build.log
+          eval "$NEXTSTRAIN_BUILD_COMMAND" |& tee "$NEXTSTRAIN_BUILD_LOG"
       # Attempt to get the AWS Batch ID even if the run build command failed
```

Comment:
Suggestion: `tee` to `.git/build.log` then move it to `build.log` after `nextstrain build` is finished. This allows presence under `.git` to be kept as an implementation detail.

Reply:
OH! Good idea! Added in https://github.com/nextstrain/.github/pull/78/commits/b9a9ae9b2aa12a5943834ea387e322db08788a73